doc/source/overview.rst

```diff
@@ -117,6 +117,30 @@ Institutional Partners
 
 The information about current institutional partners can be found on `pandas website page <https://pandas.pydata.org/about.html>`__.
 
+Modules Privacy
+----------------
+
+The following modules in pandas are considered PRIVATE:
+
+::
```

Comment:
Does the double colon format this as a code block? If so, I don't think we need that. Sphinx should render the nested list fine.

Reply:
@aparimeet can you check how this renders without the double-colons? A regular nested list should be fine.

You may need to remove the line-breaks between top-level items, and put spaces before the `-`.